triss-core/src/main/kotlin/com/pp/trisscore/service/WildaService.kt

```diff
@@ -38,7 +41,7 @@ class WildaService(val employeeService: EmployeeService,
     }
 
     private fun validateApproveAndSaveApplication(dbApplicationInfo: ApplicationInfo?, reqApplicationInfo: ApplicationInfo): Mono<out Application>? {
```

Comment:
why we are getting as parameter nullable value (?), when then we are assuming that this is not null (!!)? if this method require to this not to be null, it should be in parameter and when we are firing it we shoudl eventually use !!